browser/resources/settings/brave_overrides/personalization_options.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import {RegisterPolymerTemplateModifications} from 'chrome://brave-resources/polymer_overriding.js'
+
+import '../brave_privacy_page/brave_personalization_options.m.js'
```

Comment:
Shouldn't this be `../brave_privacy_page/brave_personalization_options.js` instead?

Reply:
We are still using `polymer_modulizer` to process `brave_personalization_options.html and .js` (see https://github.com/brave/brave-core/blob/master/browser/resources/settings/brave_privacy_page/BUILD.gn#L19) which results in the `.m.js` being generated - and that's the one we want here.

Reply:
Ah, ok. Polymer always confuses me so I was just double-checking it was not a mistake, thanks for clarifying!